protocol/contracts/beanstalk/sun/SeasonFacet/SeasonGettersFacet.sol

```diff
@@ -264,36 +264,46 @@ contract SeasonGettersFacet {
     }
 
     /**
-     * @notice returns the twa beanEth liquidity, using the values stored in beanstalk.
+     * @notice returns the twa liquidity for a well, using the values stored in beanstalk.
      */
-    function getBeanEthTwaUsdLiquidity() public view returns (uint256) {
+    function getTwaLiquidityForWell(address well) public view returns (uint256) {
+        (address token, ) = LibWell.getNonBeanTokenAndIndexFromWell(well);
         return LibWell.getTwaLiquidityFromBeanstalkPump(
-            C.BEAN_ETH_WELL,
-            LibUsdOracle.getTokenPrice(C.WETH)
+            well,
+            LibUsdOracle.getTokenPrice(token)
         );
     }
-    
+
     /**
-     * @notice returns the non-bean usd total liquidity of bean.
+     * @notice returns the twa liquidity for a well, using the values stored in beanstalk.
+     * @dev This is the liquidity used in the gauge system.
      */
-    function getTotalUsdLiquidity() external view returns (uint256) {
-        return getBeanEthTwaUsdLiquidity();
+    function getWeightedTwaLiquidityForWell(address well) public view returns (uint256) {
+        return LibEvaluate.getLiquidityWeight(s.ss[well].lwSelector)
+            .mul(getTwaLiquidityForWell(well))
+            .div(1e18);
     }
 
     /**
-     * @notice returns the weighted beanEth liquidity used in Gauge calculations.
+     * @notice Returns the total twa liquidity of beanstalk.
      */
-    function getWeightedBeanEthTwaUsdLiquidity() public view returns (uint256) {
-        return LibEvaluate.getLiquidityWeight(s.ss[C.BEAN_ETH_WELL].lwSelector)
-            .mul(getBeanEthTwaUsdLiquidity())
-            .div(1e18);
+    function getTotalUsdLiquidity() external view returns (uint256 totalLiquidity) {
```

Comment:
We currently assume all current and future liquidity pools are wells in the gauge system, and that new lp that aren't wells will require an update to the gauge system (`libEvaluate.calcLPToSupplyRatio()`). Don't think its needed to explicitly write wells because of this.